src/main/java/StudentSystem/Controller/CourseController.java

```diff
@@ -1,7 +1,9 @@
 package StudentSystem.Controller;
```

Comment:
Pro-tip: Package names are usually one word, with the first letter being lowercase.